src/Cache/AbstractHasCacheTtlInResponse.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace PodPoint\Reviews\Cache;
+
+use PodPoint\Reviews\Request\AbstractCacheableRequest;
+
+/**
+ * Class AbstractHasCacheTtlInResponse
+ */
+abstract class AbstractHasCacheTtlInResponse extends AbstractCacheableRequest
+{
+    /**
+     * The name of the field which has expires or ttl.
+     * @var
+     */
+    protected $cacheTtlResponseField;
+
+    /**
+     * @param array|null $responseBody
```

Comment:
I know it's probably a bit redundant but do you mind to add just a brief explanation